### ISSUE TITLE
Test results

### ISSUE THREAD

**Issue by AntoineDemeules** [MaybeShewill-CV/lanenet-lane-detection#226]
Hi, I tested the last pre-trained model you shared on the Tusimple dataset and the results are almost perfect. However when I test it on my own dataset the results are completely incorrect (a lot worst than with your previous pre-trained model). My images are also (1280x720). Is there any preprocessing I should add to obtain better results? Could you explain me why are you doing "image = image / 127.5 - 1.0"? Thanks.

**Comment by MaybeShewill-CV** [comment]
@AntoineDemeules To scale the image values into [-1, 1]:)

**Comment by AntoineDemeules** [comment]
Oh yeah of course, my bad. One more thing, I see that you're doing remapping using an ipm file.  Could you explain me quickly how the remapping with ipm file works please? I suppose the one you shared is only valid for Tusimple dataset. How could I create an ipm file for my own dataset?

**Comment by MaybeShewill-CV** [comment]
@AntoineDemeules You may find some uesful information [here](https://github.com/MaybeShewill-CV/Easy-Ipm-Client) :)

**Comment by AntoineDemeules** [comment]
Great, I will take a look. Thank you very much for answering really fast.

**Comment by AN-Ahmed** [comment]
@AntoineDemeules I'm currently testing the model with my own dataset that have the same dimensions. When I run the model , I get the src image with just the original image with no lanes marked on it , and the masked image is just a black image, can you please advise me what have you done in case you got results from testing the model with another image?

Thanks :)

**Comment by AntoineDemeules** [comment]
@AN-Ahmed , Sorry for the late answer. Personnaly I stayed with the previous pretrained model because it gave me better results than the new one. If nothing appears on your mask, it means that no lane is detected. Maybe your dataset is just too different from the one used to train the model. In that case there is not many things you can do except training the model yourself (if you have annoted data) or find a better pretrained model. I haven't worked on this for a long time I am not aware of the updates on the code so don't take this response to seriously.
